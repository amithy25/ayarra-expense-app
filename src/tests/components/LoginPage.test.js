import React from "react";
import { LoginPage } from "../../components/LoginPage";
import { shallow } from "enzyme";
import { startLogin } from "../../actions/auth";

test("should login ", () => {
  const wrapper = shallow(<LoginPage />);
  expect(wrapper).toMatchSnapshot();
});

test("should call start login on button click", () => {
  const startLogin = jest.fn();
  const wrapper = shallow(<LoginPage startLogin={startLogin} />);
  wrapper.find("button").simulate("click");
  expect(startLogin).toHaveBeenCalled();
});
