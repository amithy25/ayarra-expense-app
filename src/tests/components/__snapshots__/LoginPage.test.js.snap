// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should login  1`] = `
<button>
  Login
</button>
`;
